utensor_cgen/code_generator.py

```diff
@@ -48,38 +58,40 @@ def generate(self, src_fname):
 
     opFactory = OperatorFactory()
 
-    print("Transforming graph: {}".format(self.pb_file))
-    new_pb_file = self._transform_graph(self.pb_file)
-    print("Parsing new pb file {}".format(new_pb_file))
-    ops_info, topo_orders = parse_pb(new_pb_file, self.output_nodes)
-    if self.clear_tmp_file:
-      os.remove(new_pb_file)
-    construct_order = RefCntOptimizer.optimize(ops_info, topo_orders, self.output_nodes, self.method)
-
-    # TODO better snippet construction abstraction
-    for op_id, (op_name, op_info, ref_counts, to_eval) in enumerate(construct_order, 1):
+    graph_def = tf.GraphDef()
+    with tf.gfile.FastGFile(self.model_file, 'rb') as fid:
+      graph_def.ParseFromString(fid.read())
+    ugraph = uTensorGraph(graph_def, self.output_nodes)
+    print("Transforming graph: {}".format(self.model_file))
+    quant_ugraph = self._transform_graph(ugraph,
```

Comment:
I suppose there should be different quantization process for other graphs.
However, with `uTensorGraph`, I'm able to construct a `GraphDef` out of it.
So maybe we can quantize other graphs with Tensorflow quantization? Not sure.

Reply:
i think we should add a check for current quantization. 
```
if already_quantized(graph):
  print warning
  skip_quantization()
```

Reply:
Um.... Checking whether if there is node like `Quantize`/`Dequantize`?

Reply:
yes, or just making sure all tensors are uint8 or uint16

Reply:
Ok, I'll do it this night after work.

Reply:
Instead of skip quantization, I show up a warning message to the user.
Since the transformation is also configurable by the user, there is no trivial way to determine what a user want based on all possible transformation and given graph type.

It's the user's responsibility to do it right.
At least they can fix the issue by the warning message if any.

Reply:
ok, we might want to give them the option to skip quantization entirely as a flag.